### PR TITLE
add input examples to mimecast

### PR DESCRIPTION
**Pull request by mhofert-r7**
### Description

Describe the proposed changes:

add input examples to mimecast

### Testing

None documentation change only

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: mimecast/help.md [hunk 1]
```diff
@@ -158,11 +195,21 @@ This action is used to permit or block a sender.
...
+|sender|string|None|True|The email address of the external sender|None|user@example.com|
+|to|string|None|True|The email address of the internal recipient|None|user@example.com|
+
+Example input:
+
+```
+{
+  "action": "block",
+  "sender": "user@example.com",
+  "to": ""
```
Comment (jschipp-r7) [inline mimecast/help.md:210]:
```suggestion
  "to": "user@rapid7.com"
```

File: mimecast/help.md [hunk 2]
```diff
@@ -158,11 +195,21 @@ This action is used to permit or block a sender.
...
+|sender|string|None|True|The email address of the external sender|None|user@example.com|
+|to|string|None|True|The email address of the internal recipient|None|user@example.com|
+
+Example input:
+
+```
+{
+  "action": "block",
+  "sender": "user@example.com",
+  "to": "user@rapid7.com"
```
Comment (tylerslijboom-r7) [inline mimecast/help.md:210]:
```suggestion
  "to": "user@example.com"
```

File: mimecast/help.md [hunk 3]
```diff
@@ -193,18 +240,31 @@ This action is used to create a blocked sender policy.
...
+|to_type|string|everyone|True|Can be one of: everyone, internal_addresses, external_addresses, email_domain, profile_group or individual_email_address|['everyone', 'internal_addresses', 'external_addresses', 'email_domain', 'profile_group', 'individual_email_address']|everyone|
+|to_value|string|None|False|Required if `To Type` is one of email_domain, profile_group, individual_email_address. Expected values: If `To Type` is email_domain, a domain name without the @ symbol. If `To Type` is profile_group, the ID of the profile group. If `To Type` is individual_email_address, an email address|None|user@example.com|
+
+Example input:
+
+```
+{
+  "description": "komand test",
+  "from_part": "envelope_from",
+  "from_type": "email_domain",
```
Comment (tylerslijboom-r7) [inline mimecast/help.md:260]:
```suggestion
  "from_type": "internal_addresses",
```
Matching the table examples

File: mimecast/help.md (hunk 2 above)
Comment (tylerslijboom-r7) [reply mimecast/help.md:210]: Matching the table examples.

File: mimecast/help.md [hunk 4]
```diff
@@ -337,10 +425,20 @@ This action is used to find groups that match a given query.
...
+|----|----|-------|--------|-----------|----|-------|
+|query|string|None|False|A string to query for|None|mygroup|
+|source|string|cloud|True|A group source to filter on, either "cloud" or "ldap"|['cloud', 'ldap']|cloud|
+
+Example input:
+
+```
+{
+
+  "query": "exam.*",
```
Comment (tylerslijboom-r7) [inline mimecast/help.md:438]:
```suggestion
  "query": "exam.*",
```
Can this be made to match the table?

File: mimecast/help.md [hunk 5]
```diff
@@ -379,13 +477,24 @@ This action is used to get TTP URL logs.
...
+|scan_result|string|all|True|Filters logs by scan result, must be one of clean, malicious, or all|['clean', 'malicious', 'all']|malicious|
+|to|string|None|False|End date of logs to return in the following format 2015-11-16T14:49:18+0000. Default is time of request|None|2018-11-22T14:49:18+0000|
+|url_to_filter|string|None|False|Regular expression to filter on. e.g. `examp` will return only URLs with the letters examp in them|None|exam.*|
+
+Example input:
+
+```
+{
+  "from": "2019-04-23T11:00:00+0000",
+  "route": "all",
```
Comment (tylerslijboom-r7) [inline mimecast/help.md:493]:
```suggestion
  "route": "inbound",
```

Comment (tylerslijboom-r7) [reply mimecast/help.md:493]: Can this example be made to match the table?

**Pull request merged**